examples/sample-app/README.md

```diff
@@ -462,10 +462,11 @@ Another interesting example is deleting a pod.
 
 Cleaning Up
 -----------
-To clean up all of your environment, you can run the script:
+To empty your project, deleting all BuildConfigs, ImageStreams, DeploymentConfigs,
+Services, etc, you can use:
 
-        $ sudo ./cleanup.sh
+    $ oc delete all --all
 
-This will stop the `openshift` process, remove files created by OpenShift and kill all Docker containers created by Kubernetes in your host system.  The cleanup script needs root privileges to be able to remove all the directories OpenShift created.
+Alternatively, you can delete the whole project with:
```

Comment:
@kargakis I'm using [*alternatively*](http://www.thefreedictionary.com/alternatively) in the sense of "suggesting a different course of action" -- it doesn't mean that `oc delete all --all` and `oc delete project` are equivalent.
I'd expect to find more details about what each command do either in the output of `oc delete --help` and/or the docs.